posthog/models/dashboard_tile.py

```diff
@@ -107,37 +105,6 @@ def dashboard_queryset(queryset: QuerySet) -> QuerySet:
         )
 
 
-@mutable_receiver(post_save, sender=Insight)
```

Comment:
I assume you've done the thinking but how do we know these are safe to remove?

If I'm editing and moving backwards and forwards between states aren't these what let me hit the cache?

Reply:
Not sure if you mean this, but we don't read Insight/DashboardTile.filters_hash any more. InsightCachingState has its own signal handling in posthog/models/insight_caching_state.py

Reply:
I'd missed how much tidier that all is now! 🙌